infra/base-images/base-clang/Dockerfile

```diff
@@ -27,6 +27,8 @@ RUN apt-get update && apt-get install -y wget sudo && \
     rm cmake-$CMAKE_VERSION-Linux-x86_64.sh && \
     SUDO_FORCE_REMOVE=yes apt-get remove --purge -y wget sudo
 
+RUN apt-get install -y zlib1g-dev
```

Comment:
This is needed because rustc uses zlib compression for the coverage section, and llvm-cov needs to decode it